src/mod_muc_admin.erl

```diff
@@ -774,8 +774,7 @@ find_serverhost(Host, ServerHosts) ->
 act_on_room(destroy, {N, H, Pid}, SH) ->
     p1_fsm:send_all_state_event(
       Pid, {destroy, <<"Room destroyed by rooms_unused_destroy.">>}),
-    mod_muc:room_destroyed(H, N, Pid, SH),
-    mod_muc:forget_room(SH, H, N);
```

Comment:
Isn't this actually the meat of the change?  not sure what's going on above, and again I'd put a comment in.

Reply:
Its all meat.  This is the part where we don't remove the room from mam, only from the muc_online_rooms in-memory table.